.github/workflows/notebook-interfaces-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Notebooks: interfaces
+
+on:
+  schedule:
+    # 4am daily
+    - cron:  '0 4 * * *'
+
+jobs:
+
+  build-and-test:
+    name: notebooks
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: install pints
+        run: |
+          python --version
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install .
+          python -m pip install .[dev]
+
+      - name: run jupyter notebooks
+        run: |
+          python -m pip install statsmodels
+          python run-tests.py --books
```

Comment:
this will run all the notebooks right? I think you want to just run the interface ones

Reply:
@chonlei Thanks -- have made those changes.

Reply:
I didn't make the change to the parameter names in tables because the latex didn't render...

Reply:
Hmm, also, shouldn't we take out the interface ones from the --books? Afraid this will break the `.github/workflows/notebook-test.yml` test...

Reply:
Should be fixed now.